### PR TITLE
Feature/tranql standalone

### PR DESCRIPTION
**Pull request by YaphetKG**

### CODE REVIEW

File: helx/charts/search/values.yaml
Comment (cschreep) [inline helx/charts/search/values.yaml:143]: I wonder if we should leave this as master, since that is the "released" version of the dags. It can always be overriden on a per-environment basis though

File: helx/charts/search/values.yaml
Comment (cschreep) [inline helx/charts/search/values.yaml:334]: Do the redis slaves support write access? or will it only be used for reads?

**Review by cschreep — APPROVED** [review]
A couple questions but this is a helpful improvement

File: helx/charts/search/values.yaml
Comment (YaphetKG) [reply helx/charts/search/values.yaml:334]: tranql does only reads.

File: helx/charts/search/values.yaml
Comment (YaphetKG) [reply helx/charts/search/values.yaml]: makes sense, Changed back to master (was just holding off till we have airflow 2.0 compatible release for roger).

**Review by cschreep — APPROVED** [review]
👍🏻  looks good

**Pull request merged**
